<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntersectionGUIClass</class>
 <widget class="QMainWindow" name="IntersectionGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IntersectionGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pointNumResult">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;Answer:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="allShapes">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>251</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getResult">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>求解</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteShape">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除图形</string>
    </property>
   </widget>
   <widget class="QWidget" name="midShow" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>39</y>
      <width>481</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_in">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_out">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
